specs-go/state.go

```diff
@@ -1,5 +1,20 @@
 package specs
 
+const (
+	// StateCreating indicates that the container is being created
+	StateCreating = "creating"
+
+	// StateCreated indicates that the runtime has finished the create operation
+	StateCreated = "created"
+
+	// StateRunning indicates that the container process has executed the
+	// user-specified program but has not exited
+	StateRunning = "running"
+
+	// StateStopped indicates that the container process has exited
+	StateStopped = "stopped"
+)
```

Comment:
can we add some comments about the delta between these and runc status states:
```
func (s Status) String() string {
	switch s {
	case Created:
		return "created"
	case Running:
		return "running"
	case Pausing:
		return "pausing"
	case Paused:
		return "paused"
	case Stopped:
		return "stopped"
	default:
		return "unknown"
	}
}
```
Creating being new.. (and not yet discussed in the spec?)  It needs a bit of an intro maybe? 

The missing ones caught my eye.. but I get that this is the runtime-spec and the other states are not yet covered.. maybe a TODO for pulling them in?

Reply:
The pause-related ones aren't in the runtime-spec because we don't have a pause operation in the spec (among other missing things, such as the lack of an exec operation). I'd say that we should only define the ones which are actually listed in the spec and in `runc` we can just add the ones we have.

Reply:
agreed...shouldn't introduce new state values that are not in the spec. Might be better just to leave them all out..

Reply:
So I suppose this is resolved?

Reply:
need to remove ```StateCreating = "creating"```

Reply:
I thought I had left a comment here, it looks like it probably didn't get sent :D ?
Anyways, I just double checked, this enum has all the states defined in the [spec here](https://github.com/opencontainers/runtime-spec/blob/master/runtime.md)

Reply:
oh man.. I entirely missed that a new quasi state was added for between the create request and actual created.